src/test/regress/expected/distributed_collations_conflict.out

```diff
@@ -8,6 +8,7 @@ SELECT run_command_on_workers($$CREATE SCHEMA collation_conflict;$$);
 
 \c - - - :worker_1_port
 SET search_path TO collation_conflict;
+SET citus.enable_metadata_sync TO off;
```

Comment:
Can you add comment why we are disabling the metadata sync here (you may add https://github.com/citusdata/citus/issues/5583 like we do within the schedule right now)?

Reply:
That's not the reason actually. As we used to do `citus.enable_object_propagation TO OFF` all over the tests from the workers, now we need to do this.

I think as readers realize  the meaning of ` citus.enable_object_propagation ` such a comment would be redundant, so I skip if you are OK :)